src/main.rs

```diff
@@ -191,7 +211,7 @@ fn main() -> Result<(), lexopt::Error> {
     }
 
     let config = {
-        if let Ok(mut f) = File::options().read(true).open(CONFIG_FILE_NAME) {
+        if let Ok(mut f) = File::options().read(true).open(config_file) {
```

Comment:
CONFIG_FILE_NAME became config_file? What happened there, this seems unrelated

Reply:
It would be better in another PR since this will be in conflict with two PR I just merged ;) ( typstfmt-config -> .typstfmt && a global option, this would also be a very nice feature tho !)